packages/db/src/util.ts

```diff
@@ -0,0 +1,12 @@
+import {Bucket} from ".";
+
+export function getBucketNameByValue<T extends Bucket>(enumValue: T): keyof typeof Bucket | null {
+  const keys = Object.keys(Bucket).filter((x) => {
+    if (isNaN(parseInt(x))) {
+      return Bucket[x as keyof typeof Bucket] == enumValue;
+    } else {
+      return false;
+    }
+  }) as (keyof typeof Bucket)[];
+  return keys.length > 0 ? keys[0] : null;
+}
```

Comment:
Should we throw here instead of returning null? If for some reason this function returns null the metrics will break and it won't be immediately obvious why. Adding a unit test should help ensure that it doesn't break on production first.

Reply:
I should test it but my understanding is that it will only go in metrics with no bucket label